### PR TITLE
Add in TP bow and simplify commands to one

### PR DESCRIPTION
**Pull request by notnotnotswipez**
Removed the permission check in the plugin.yml file, and replaced it with a central command in the main command. Permission is "BowTP.main". Also added in a Teleportation bow, meaning only the teleportation bow will make you teleport instead of every bow

### CODE REVIEW

**Comment by cojored** [comment]
I need to figure out how to just accept the command changes I want the bow things to stay the same I will get back to this when I get a chance to check how to just accept the changes for the commands

File: src/main/java/com/heppcat/items/TPBowItem.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import org.bukkit.inventory.meta.ItemMeta;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TPBowItem {
+
+    public static ItemStack tpbow;
+
+    public static void Init(){
```
Comment (abidaJustice) [inline src/main/java/com/heppcat/items/TPBowItem.java:17]: I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Comment (notnotnotswipez) [reply src/main/java/com/heppcat/items/TPBowItem.java:17]: The Init method? Its one word, its usually uppercase anyways xd. Also, im not sure if you know this, but its a suggestion, not a requirement

Comment (cojored) [reply src/main/java/com/heppcat/items/TPBowItem.java:17]: Yeah it is definitely a suggestion and not a requirement

**Pull request closed without merging**